pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
```

Comment:
I'm OK with copying them.

> The only thing that really needs to happen is for the destination directory to contain something, created by the backend, that the backend itself knows how to turn it into a wheel

That's a good description of what I was trying to get at. There may be other use cases (building sdists being the obvious one!) so I'd state it a little more generically as "the hook should write to the destination directory the minimal set of files needed by the backend, that the backend can use to build a wheel". But that's just details.